Intersect.Server/Migrations/Logging/LoggingContextModelSnapshot.cs

```diff
@@ -16,6 +16,30 @@ protected override void BuildModel(ModelBuilder modelBuilder)
             modelBuilder
                 .HasAnnotation("ProductVersion", "2.1.3-rtm-32065");
 
+            modelBuilder.Entity("Intersect.Server.Database.Logging.Entities.ChatHistory", b =>
+                {
+                    b.Property<Guid>("Id")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("Ip");
+
+                    b.Property<string>("MessageText");
+
+                    b.Property<int>("MessageType");
+
+                    b.Property<Guid?>("PlayerId");
+
+                    b.Property<string>("PlayerName");
+
+                    b.Property<DateTime>("TimeStamp");
+
+                    b.Property<Guid?>("UserId");
```

Comment:
When you regenerate the migration after correcting the models this should be automatically corrected to `Guid` and won't need manual intervention.